extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -159,6 +159,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_.addEventListener(
           'touchend', this.touchEndHandler_.bind(this));
     }
+
+    this.registerAction('moveToSlideIndex', invocation => {
```

Comment:
@choumx - i would want it to be consistent with yours, also i want it to be explicit and say include the word `index` let me know if you can comeup with something for me :)